### ISSUE TITLE
Build failure on ios

### ISSUE THREAD

**Issue by Toolenaar** [RevenueCat/purchases-flutter#122]
I have similar build issues as other posters. But I have tried everything to fix it to no avail. I dont know if I am missing something or if this is just a different issue.

I am on v ^1.4.1 of the plugin

1. Environment
    1. Platform: iOS
    2. SDK version: Channel stable, 1.22.2
    3. OS version: 12
    4. XCode: 12.1

## build log

**Additional context**
Xcode build done.                                           384.6s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1462:38: warning: variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1426:3: note: variable 'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1462:48: warning: variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1425:3: note: variable 'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1546:38: warning: variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1510:3: note: variable 'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1546:48: warning: variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/dsp/dec_neon.c:1509:3: note: variable 'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    4 warnings generated.
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/enc/backward_references_enc.c:663:12: warning: variable 'best_offset' may be uninitialized when used here [-Wconditional-uninitialized]
              (best_offset << MAX_LENGTH_BITS) | (uint32_t)best_length;
               ^~~~~~~~~~~
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/libwebp/src/enc/backward_references_enc.c:595:20: note: initialize the variable 'best_offset' to silence this warning
        int best_offset;
                       ^
                        = 0
    1 warning generated.
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: warning: coercion of implicitly unwrappable value of type 'InAppWebView_IBWrapper?' to 'Any' does not unwrap optional
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:26:19: note: implicitly unwrapped property 'containerWebView' declared here
        @IBOutlet var containerWebView: InAppWebView_IBWrapper!
                      ^
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: provide a default value to avoid this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                       ?? <#default value#>
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: force-unwrap the value to avoid this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                      !
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                       as Any
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:13:17: warning: method definition for '_setPushTokenString:' not found [-Wincomplete-implementation]
    @implementation RCPurchases (HybridAdditions)
                    ^
    In file included from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:6:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.h:14:1: note: method '_setPushTokenString:' declared here
    - (void)_setPushTokenString:(nullable NSString *)pushToken;
    ^
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:13:17: warning: method definition for
    'configureWithAPIKey:appUserID:observerMode:userDefaults:platformFlavor:platformFlavorVersion:' not found [-Wincomplete-implementation]
    @implementation RCPurchases (HybridAdditions)
                    ^
    In file included from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:6:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.h:23:1: note: method 'configureWithAPIKey:appUserID:observerMode:userDefaults:platformFlavor:platformFlavorVersion:'
    declared here
    + (instancetype)configureWithAPIKey:(NSString *)APIKey
    ^
    2 warnings generated.
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/purchases_flutter-1.4.1/ios/Classes/PurchasesFlutterPlugin.m:197:28: warning: 'addAttributionData:network:networkUserId:' is deprecated: Use the set<NetworkId> functions instead. [-Wdeprecated-declarations]
        [RCCommonFunctionality addAttributionData:data network:network networkUserId:networkUserId];
                               ^
    In module 'PurchasesHybridCommon' imported from /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/purchases_flutter-1.4.1/ios/Classes/PurchasesFlutterPlugin.m:5:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCCommonFunctionality.h:22:130: note: 'addAttributionData:network:networkUserId:' has been explicitly marked deprecated here
    + (void)addAttributionData:(NSDictionary *)data network:(NSInteger)network networkUserId:(NSString *) networkUserId __attribute((deprecated("Use the set<NetworkId> functions instead.")));
                                                                                                                                     ^
    1 warning generated.
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/FIRMessagingPubSub.m:283:12: warning: parameter 'expression' not found in the function declaration [-Wdocumentation]
     *  @param expression The topic expression used to generate a pubsub topic
               ^~~~~~~~~~
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/FIRMessagingPubSub.m:283:12: note: did you mean 'topic'?
     *  @param expression The topic expression used to generate a pubsub topic
               ^~~~~~~~~~
               topic
    1 warning generated.
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/firebase_remote_config-0.4.0/ios/Classes/FirebaseRemoteConfigPlugin.m:44:65: warning: 'isDeveloperModeEnabled' is deprecated: This no longer needs to be set during development. Refer to documentation for additional details.
    [-Wdeprecated-declarations]
            [[NSNumber alloc] initWithBool:[firRemoteConfigSettings isDeveloperModeEnabled]];
                                                                    ^
    In module 'FirebaseRemoteConfig' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Headers/Public/Firebase/Firebase.h:146:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h:141:37: note: property 'isDeveloperModeEnabled' is declared deprecated here
    @property(nonatomic, readonly) BOOL isDeveloperModeEnabled DEPRECATED_MSG_ATTRIBUTE(
                                        ^
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h:141:60: note: 'isDeveloperModeEnabled' has been explicitly marked deprecated here
    @property(nonatomic, readonly) BOOL isDeveloperModeEnabled DEPRECATED_MSG_ATTRIBUTE(
                                                               ^
    In module 'UIKit' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Target Support Files/firebase_remote_config/firebase_remote_config-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/usr/include/AvailabilityMacros.h:181:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/firebase_remote_config-0.4.0/ios/Classes/FirebaseRemoteConfigPlugin.m:53:42: warning: 'initWithDeveloperModeEnabled:' is deprecated: This no longer needs to be set during development. Refer to documentation for additional details.
    [-Wdeprecated-declarations]
            [[FIRRemoteConfigSettings alloc] initWithDeveloperModeEnabled:debugMode];
                                             ^
    In module 'FirebaseRemoteConfig' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Headers/Public/Firebase/Firebase.h:146:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h:148:5: note: 'initWithDeveloperModeEnabled:' has been explicitly marked deprecated here
        DEPRECATED_MSG_ATTRIBUTE("This no longer needs to be set during development. Refer to "
        ^
    In module 'UIKit' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Target Support Files/firebase_remote_config/firebase_remote_config-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/usr/include/AvailabilityMacros.h:181:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/firebase_remote_config-0.4.0/ios/Classes/FirebaseRemoteConfigPlugin.m:100:54: warning: 'activateFetched' is deprecated: Use -[FIRRemoteConfig activate] instead. [-Wdeprecated-declarations]
        BOOL newConfig = [[FIRRemoteConfig remoteConfig] activateFetched];
                                                         ^
    In module 'FirebaseRemoteConfig' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Headers/Public/Firebase/Firebase.h:146:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h:252:25: note: 'activateFetched' has been explicitly marked deprecated here
    - (BOOL)activateFetched DEPRECATED_MSG_ATTRIBUTE("Use -[FIRRemoteConfig activate] instead.");
                            ^
    In module 'UIKit' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Target Support Files/firebase_remote_config/firebase_remote_config-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/usr/include/AvailabilityMacros.h:181:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/firebase_remote_config-0.4.0/ios/Classes/FirebaseRemoteConfigPlugin.m:131:40: warning: 'allKeysFromSource:namespace:' is deprecated: Use -[FIRRemoteConfig allKeysFromSource:] instead. [-Wdeprecated-declarations]
      NSArray *defaultKeys = [remoteConfig allKeysFromSource:FIRRemoteConfigSourceDefault
                                           ^
    In module 'FirebaseRemoteConfig' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Headers/Public/Firebase/Firebase.h:146:
    /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h:304:5: note: 'allKeysFromSource:namespace:' has been explicitly marked deprecated here
        DEPRECATED_MSG_ATTRIBUTE("Use -[FIRRemoteConfig allKeysFromSource:] instead.");
        ^
    In module 'UIKit' imported from /Users/jochemtoolenaar/Documents/src/defigners/defigned/streeet/streeet/ios/Pods/Target Support Files/firebase_remote_config/firebase_remote_config-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.1.sdk/usr/include/AvailabilityMacros.h:181:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    4 warnings generated.
    6 warnings generated.
    /Users/jochemtoolenaar/.pub-cache/hosted/pub.dartlang.org/cloud_functions-0.6.0/ios/Classes/CloudFunctionsPlugin.m:54:59: warning: comparison of distinct pointer types ('NSNumber *' and 'NSNull * _Nonnull') [-Wcompare-distinct-pointer-types]
        if (timeoutMicroseconds != nil && timeoutMicroseconds != [NSNull null]) {
                                          ~~~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~
    1 warning generated.

    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'AppAuth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'connectivity' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'cloud_functions' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_remote_config' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'cloud_firestore' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'webview_flutter' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'wakelock' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'video_thumbnail' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'video_player' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'url_launcher' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'sqflite' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'shared_preferences' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'share' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'screen' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'path_provider' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'package_info' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'nanopb' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'location' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'libwebp' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'leveldb-library' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'image_picker' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'geolocator' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'google_sign_in' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'gRPC-Core' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'geocoding' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'gRPC-C++-gRPCCertificates-Cpp' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'gRPC-C++' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'flutter_inappwebview' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_messaging' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_dynamic_links' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_storage' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_crashlytics' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_core' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_analytics' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'firebase_auth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'device_info' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'apple_sign_in' from project 'Pods')
    warning: The use of Swift 3 @objc inference in Swift 4 mode is deprecated. Please address deprecated @objc inference warnings, test your code with “Use of deprecated Swift 3 @objc inference” logging enabled, and then disable inference by changing the "Swift 3 @objc Inference" build setting to
    "Default" for the "Runner" target. (in target 'Runner' from project 'Runner')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'abseil' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'Reachability' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'Protobuf' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'GoogleUtilities' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'GoogleDataTransportCCTSupport' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'GoogleDataTransport' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'GTMAppAuth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseStorage' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseRemoteConfig' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseMessaging' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseInstanceID' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseInstallations' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseFunctions' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseFirestore' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseDynamicLinks' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseCoreDiagnostics' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseCrashlytics' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseCore' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseAuth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseABTesting' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FMDB' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FBSDKCoreKit' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'BoringSSL-GRPC' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FBSDKLoginKit' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'GoogleSignIn' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseAnalyticsInterop' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'Flutter' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseAnalytics' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseCoreDiagnosticsInterop' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'Firebase' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'FirebaseAuthInterop' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'GoogleAppMeasurement' from project 'Pods')

**Comment by aboedo** [comment]
@Toolenaar I can't find any concrete errors on those logs. I think the actual build failure reason is missing. Could you search for it and let us know what it is? 

As a side note, I do see that the deployment is set to 8.0 (which is the default on cocoapods), so you may have missed [the step about setting it to at least 9.0](https://docs.revenuecat.com/docs/flutter#ios-deployment-target)

**Comment by abaturyna** [comment]
I am facing the same error!!!!!!!!!! 
My deployement target is 10.0 . There are my logs : 

    error: the following command failed with exit code 0 but produced no further output
    CompileC /Users/nastya/Library/Developer/Xcode/DerivedData/Runner-eqykqzahaywvspegnwfxiugoiwez/Build/Intermediates.noindex/Pods.build/Release-iphoneos/purchases_flutter.build/Objects-normal/arm64/PurchasesFlutterPlugin.o
    /Users/nastya/Dropbox/BUSINESS/Soft/flutter/.pub-cache/hosted/pub.dartlang.org/purchases_flutter-1.4.1/ios/Classes/PurchasesFlutterPlugin.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    Undefined symbols for architecture armv7:
      "_OBJC_CLASS_$_RCPurchases", referenced from:
          objc-class-ref in PurchasesFlutterPlugin.o
    ld: symbol(s) not found for architecture armv7
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    Undefined symbols for architecture arm64:
      "_OBJC_CLASS_$_RCPurchases", referenced from:
          objc-class-ref in PurchasesFlutterPlugin.o
    ld: symbol(s) not found for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

**Comment by aboedo** [comment]
I'm closing this ticket as a duplicate of #120 
Please try following the steps in that ticket to get it fixed.
